sonar-helm-for-iac/build.gradle.kts

```diff
@@ -91,7 +93,32 @@ if (!isCi) {
 
         inputs.file("Dockerfile")
 
-        commandLine("docker", "buildx", "build", "--platform", "linux/amd64", "-t", "sonar-iac-helm-builder", "--progress", "plain", "${project.projectDir}")
+        val uidProvider = objects.property<Long>()
+        val os = DefaultNativePlatform.getCurrentOperatingSystem()
+        if (os.isLinux || os.isMacOsX) {
+            // UID of the user inside the container should match this of the host user, otherwise files from the host will be not accessible by the container.
+            val uid = com.sun.security.auth.module.UnixSystem().uid
+            uidProvider.set(uid)
+        }
+
+        val arguments = buildList {
+            add("docker")
+            add("buildx")
+            add("build")
+            if (uidProvider.isPresent) {
+                add("--build-arg")
+                add("UID=${uidProvider.get()}")
+            }
+            add("--platform")
+            add("linux/amd64")
+            add("-t")
+            add("sonar-iac-helm-builder")
+            add("--progress")
+            add("plain")
+            add("${project.projectDir}")
+        }
+
+        commandLine(arguments)
```

Comment:
We can add `setErrorOutput(System.out)` here and in `compileGoCode` for clean output